hw/ip/rstmgr/data/rstmgr.hjson.tpl

```diff
@@ -360,7 +360,8 @@
         ''',
         count: "NumSwResets",
         swaccess: "rw0c",
-        hwaccess: "hro",
+        hwaccess: "none",
```

Comment:
Weird, doesn't hardware need to read this to determine if updates to SW_RST_CTRL_N are allowed?